packages/appstore/src/assets/svgs/wallets/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import upgrade_desktop from 'Assets/svgs/wallets/wallets-upgrade-desktop.svg';
+import upgrade_mobile from 'Assets/svgs/wallets/wallets-upgrade-mobile.svg';
+import { TImageTestID, TWalletsImagesListKeys, WalletsImageProps } from './image-types';
+
+export const WalletsImagesList = {
+    upgrade_desktop,
+    upgrade_mobile,
+};
+
+const WalletsImage = ({ image, className, width }: WalletsImageProps<TWalletsImagesListKeys>) => {
+    const Component = WalletsImagesList[image] as React.ElementType;
```

Comment:
```suggestion
    const Component = WalletsImagesList[image];
```

If you do it like the above suggestions, you dont need to explicitly type them

Reply:
@yashim-deriv It doesn't work when I do like you wrote above